package-fireproof-core.json

```
@@ -60,10 +60,12 @@
   },
   "devDependencies": {},
   "peerDependencies": {
-    "@adviser/cement": "~0.3.12!",
+    "@adviser/cement": "~0.3.19!",
     "react": ">=18.0.0!"
   },
   "dependencies": {
+    "@adviser/cement": "from-package-json",
+    "@fireproof/vendor": "from-package-json",
```

Biome (1.9.4)

[error] The key @fireproof/vendor was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)